fastai/models/inceptionv4.py

```diff
@@ -5,15 +5,15 @@
 import sys
 
 model_urls = {
-    'imagenet': 'http://webia.lip6.fr/~cadene/Downloads/inceptionv4-97ef9c30.pth'
+    'inceptionv4': 'https://s3.amazonaws.com/pytorch/models/inceptionv4-58153ba9.pth'
 }
 
 class BasicConv2d(nn.Module):
 
     def __init__(self, in_planes, out_planes, kernel_size, stride, padding=0):
         super(BasicConv2d, self).__init__()
         self.conv = nn.Conv2d(in_planes, out_planes, kernel_size=kernel_size, stride=stride, padding=padding, bias=False) # verify bias false
-        self.bn = nn.BatchNorm2d(out_planes)
+        self.bn = nn.BatchNorm2d(out_planes, eps=0.001, momentum=0, affine=True)
```

Comment:
Ouch just noticed this 0 momentum=0. That means that it *never* updates the bn statistics, right? I can't imagine why that would be a good thing - will mean you can't fine-tune as well on datasets where the new domain is very different to imagenet, like the Planet dataset.

Maybe set it to 0.9? And test fine-tuning on planet?